packages/libs/eda/src/lib/core/components/visualizations/implementations/LineplotVisualization.tsx

```diff
@@ -1012,8 +1024,7 @@ function LineplotViz(props: VisualizationProps<Options>) {
       {isFaceted(data.value?.dataSetProcess) ? (
         <FacetedLinePlot
           data={
-            (vizConfig.independentAxisLogScale && vizConfig.useBinning) ||
-            (vizConfig.dependentAxisLogScale && vizConfig.showErrorBars)
+            showIndependentAxisBanner || showDependentAxisBanner
```

Comment:
@bobular Yes you are correct! data prop can be simplified with your suggestion as we have the same condition at data request. 

By the way I am afraid that I cannot understand your question. With this PR, a data request won't happen to the backend when banner is shown, because we have break condition inside data request. I have tested it again but no request is made: checked via dev tool at the browser.

Reply:
It's fine - my question was talking about the previous state of your PR but I used the future tense. Made sense only to me. My bad!